### PR TITLE
Add external links to const.

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Consolidates any external link into a `const` so we can edit it easier later.
- Replaces `<a href>` with `ExternalLink` for the external links.
- IntelliJ likes to tune in and correct some spelling in some comments 🤷🏼‍♂️

### CODE REVIEW

File: src/custom/pages/Claim/InvestmentFlow/index.tsx [hunk 1]
```diff
@@ -177,11 +178,11 @@ export default function InvestmentFlow({ hasClaims, isAirdropOnly, ...tokenAppro
           <br />
           <br />
           For more details around the token, please read{' '}
-          <a href="https://cow.fi" target="_blank" rel="noreferrer">
+          <a href={COW_LINKS.vCowPost} target="_blank" rel="noreferrer">
             the blog post
           </a>
           .<br /> For more details about the claiming process, please read the{' '}
-          <a href="https://cow.fi" target="_blank" rel="noreferrer">
+          <a href={COW_LINKS.stepGuide} target="_blank" rel="noreferrer">
```
Comment (anxolin) [inline src/custom/pages/Claim/InvestmentFlow/index.tsx:185]: not from this PR, but should be this a `ExternalLink` so we track it's click in analytics?

**Review by anxolin — APPROVED** [review]

File: src/custom/pages/Claim/InvestmentFlow/index.tsx (hunk 1 above)
Comment (biocom) [reply src/custom/pages/Claim/InvestmentFlow/index.tsx:185]: @anxolin Wasn't aware that we attached GA events to the ExternalLink, but inspected the component now and that's great. I think this PR is small so I will refactor already here.

Comment (biocom) [reply src/custom/pages/Claim/InvestmentFlow/index.tsx]: Now pushed, thank you for the suggestion.

File: src/custom/pages/Claim/index.tsx [hunk 2]
```diff
@@ -33,6 +33,12 @@ import FooterNavButtons from './FooterNavButtons'
 const GNO_CLAIM_APPROVE_MESSAGE = 'Approving GNO for investing in vCOW'
 const USDC_CLAIM_APPROVE_MESSAGE = 'Approving USDC for investing in vCOW'
 
+export const COW_LINKS = {
```
Comment (alfetopito) [inline src/custom/pages/Claim/index.tsx:36]: Can you leave a TODO here to remember to update those links?

File: src/custom/pages/Claim/index.tsx [hunk 3]
```diff
@@ -33,6 +33,12 @@ import FooterNavButtons from './FooterNavButtons'
...
+  home: 'https://cow.fi/',
```
Comment (alfetopito) [inline src/custom/pages/Claim/index.tsx:37]: Is this used anywhere?

**Review by alfetopito — APPROVED** [review]

File: src/custom/pages/Claim/index.tsx (hunk 3 above)
Comment (biocom) [reply src/custom/pages/Claim/index.tsx:37]: Actually not...will remove.

**Comment by biocom** [comment]
Addressed @alfetopito latest comments.

File: src/custom/pages/Claim/InvestmentFlow/index.tsx (hunk 1 above)
Comment (anxolin) [reply src/custom/pages/Claim/InvestmentFlow/index.tsx]: nice!

**Pull request merged**